content/guides/references/configuration.md

```diff
@@ -329,17 +329,21 @@ configuration:
 
 - `animationDistanceThreshold`
 - `baseUrl`
-- `browser` **note:** filters whether the tests or a suite of tests runs
```

Comment:
You are correct that this value cannot be updated , however this is a unique configuration option since it does allow the user to filter if their test should run. For example:
```
if('my test only for chrome', { browser: 'chrome' }, () => {})
```
Might be worth break this into its own section about filtering tests. The above example is used in quite a few places where we describe the test config concept.